creation/web_base/logging_utils.source

```diff
@@ -162,13 +164,13 @@ log_init() {
     fi
 
     # Setup tokens to authenticate with log servers
-    log_recipients=($(gconfig_get GLIDEIN_LOG_RECIPIENTS "${glidein_config}"))
+    log_recipients=($(gconfig_get GLIDEIN_LOG_RECIPIENTS_FACTORY "${glidein_config}") $(gconfig_get GLIDEIN_LOG_RECIPIENTS "${glidein_config}"))
     local no_send=0
     if [ "${#log_recipients[@]}" -eq 0 ]; then
-        warn "log_init: no recipients configured. Logs will still be produced, but not forwarded to remote servers."
+        warn "glog_init: no recipients configured. Logs will still be produced, but not forwarded to remote servers."
         no_send=1
-    elif ! log_init_tokens "${log_recipients[@]}"; then
-        warn "log_init: error while initializing tokens. Logs will still be produced, but not forwarded to remote servers."
+    elif ! _glog_log_init_tokens "${log_recipients[@]}"; then
```

Comment:
Depending on what your answer is for my comment on line 80, this needs to be changed accordingly.

Reply:
is `_glog_init_tokens`, fixed